### ISSUE TITLE
New "disable" option which turns off SMTPAutoTLS on phpmailer

### ISSUE THREAD

**Issue by slaFFik** [awesomemotive/wp-mail-smtp#22]
Based on #3.

This should fix the issue mentioned in https://wordpress.org/support/topic/smtpautotls-overrides-smtpsecure-no-tlsssl-in-php-56/

**Comment by jaredatch** [comment]
I agree with what Callum said in that thread. I think it would need to be a new specific option for it, for backwards compatibility purposes.

**Comment by slaFFik** [comment]
But it's actually a bug, that if No encryption is set, `SMTPAutoTLS` is truethy.

**Comment by slaFFik** [comment]
@jaredatch I think we should merge those No encryption and Disable.
Take a look:
![image](https://user-images.githubusercontent.com/282234/31276582-4b6086d0-aaa5-11e7-84f1-545887d312db.png)

It's just completely not clear what is the difference. No Enc is actually not SSL and not TLS, it's the same as Disable both of that. I predict, that having `SMTPAutoTLS` turned on by default as it's currently is done makes email sending process longer.
It's used in PHPMailer source coed in 1 place only `wp-includes/class-phpmailer.php:1674`), take a look:

```php
if ($this->SMTPAutoTLS and $sslext and $secure != 'ssl' and $this->smtp->getServerExt('STARTTLS')) {
    $tls = true;
}
```

So it's part of setting up a TLS negotiation.

**Comment by slaFFik** [comment]
@jaredatch And again, from the latest: https://wordpress.org/support/topic/fix-cant-connect-to-smtp/

**Comment by slaFFik** [comment]
One more related: https://wordpress.org/support/topic/plugin-not-working-with-sendpulse/ - fails after starting TLS, I assume because of that property ^.

**Comment by slaFFik** [comment]
`if` breakdown, where the `SMTPAutoTLS` is used:

```php
if ($this->SMTPAutoTLS    and    $sslext          and    $secure != 'ssl'   and     $this->smtp->getServerExt('STARTTLS')) {
    always true                  openssl algo            true if "no enc"           is STARTTLS availale on a server?
                                 since 5.0.0
```

Thus, it's safe to say, that taking into account that not only SSL option is available for users, but also No Encryption, we can safely set this var `SMTPAutoTLS = false` in that cases to prevent problems of auto TLS activation even when users set No enc.
